src/devices/Bno055/samples/README.md

```diff
@@ -0,0 +1,5 @@
+# Bno055 sensor example
+
+Please refer to [main documentation](../README.md) for more details.
+
+When using this sensor with an I2C board, just make sure you connect your I2C pins correctly as well as the alimentation adn ground.
```

Comment:
what do you mean by `alimentation`? Also type in `and` (second word from the last)